evap/student/templates/student_index.html

```diff
@@ -66,9 +69,6 @@
                                                 <span class="text text-primary" data-toggle="tooltip" data-placement="top" title="{% trans 'You did not evaluate this course.' %}"><span class="far fa-frown" aria-hidden="true"></span></span>
                                         {% endif %}
                                     </td>
-                                    <td>
-                                        {{ course.student_state|studentstatename }}
-                                    </td>
```

Comment:
there is no "Evaluated" button. there is a disabled "Evaluate" button until the evaluation period starts. then it's enabled and removed once the student evaluated the course.

i can add the gray result bar for unpublished courses, but the result bar tooltip text would have to be changed (adds an additional variable for the template include), because "Not enough answers were given." does not apply here. also, students wouldn't have a visual separation between published courses where they can't see the results and courses they already voted for where results might become available in the future. they could only figure this out by parsing the evaluation period.
it would be better to have a clear visual difference between these two states "you cannot see results because we only got little feedback" and "you cannot see results because the evaluation is still running" - we can't really use the exact same ui element. maybe another icon would work.
what's your opinion?

Reply:
we could also make a clear separation between running evaluations and all the other courses by, e.g.,  highlighting them.

there are multiple options for ordering the courses on the student index - each will probably have slightly different requirements for the results bar/highlighting/whatever. so we should decide that first:
- currently due evaluations (courses in the evaluation period where the user can still vote) should probably be on top.
- published results could be on the bottom.
- the remaining courses in between can be divided into three groups: 1. courses in preparation, 2. courses in evaluation for which the user already voted and 3. courses where the evaluation ended but which are not yet published. which order should we use for these? 1, 2, 3?

we could also use a strictly time-linear order: courses in preparation, due courses in evaluation, others courses in evaluation, unpublished, published.

what would you prefer?

Reply:
and another question: in which states should we display the "will start/end in n days/tomorrow/..." badges?

Reply:
and in which states do we want to show the participation bar?

Reply:
rabbit hole. this is more an overall student index redesign. new issue for that?

Reply:
yeah, okay. in that case i would however prefer to have no changes at all to the student pages in this branch.